tools/AnalyzerCodeGenerator/AnalyzerGenerator.Utilities/Utilities.cs

```diff
@@ -451,7 +451,7 @@ public static IEnumerable<string> GetCategories(IEnumerable<CheckData> checks)
                 "install.ps1",
                 "uninstall.ps1",
                 "packages.config",
-                "ThirdPartyNotices.rtf"
+                "ThirdPartyNoticestxt"
```

Comment:
I think we are missing a period here

Reply:
@jmarolf Is this whole project actually used?

Reply:
This does appear to be an old tool and I am not aware of the history here @mavasani do you remember what this was for?

Reply:
Can’t recall the purpose of this tool, but certainly it is no longer used.

Reply:
If you want to proceed with merging this PR, I will create another one where I remove the project. Shall it be the project only or the full AnalyzerGenerator solution?

Reply:
See #5741